Feat: Added Konnect examples for Auth plugins related to Consumers (#6742)
Kong/docs.konghq.com: app/_hub/kong-inc/acl/how-to/_index.md

* `CONSUMER`: The `id` or `username` property of the Consumer entity to associate the credentials to.
* `group`: The arbitrary group name to associate to the Consumer.
* `tags`: Optional descriptor tags for the group.
{% endnavtab %}
{% navtab Konnect %}

> After you have added an authentication plugin to a Service or a Route, and you have
> created your [consumers](/gateway/latest/admin-api/#consumer-object), you can now
> associate a group to a consumer using the following request:

Reviewer: ```suggestion
After you have added an authentication plugin to a service or a route, and you have
created the [consumers object](/gateway/latest/admin-api/#consumer-object), you can now
associate a group to a consumer using the following request:
```

```bash

> curl -X POST https://{us|eu}.api.konghq.com/v2/control-planes/{controlPlaneId}/core-entities/consumers/{CONSUMER}/acls \

Reviewer: ```suggestion
curl -X POST https://{us|eu}.api.konghq.com/v2/control-planes/{controlPlaneId}/core-entities/consumers/{CONSUMER_ID}/acls \
```

    --data "group=group1" \
    --data "tags[]=tag1" \
    --data "tags[]=tag2"
```

> `CONSUMER`: The `username` property of the consumer entity to associate the credentials with.

Reviewer: ```suggestion
`CONSUMER_ID`: The UUID of the consumer entity to associate the credentials with.
```
